documentation/leo/03_language.md

```diff
@@ -267,8 +271,10 @@ transition foo(public a: field) -> field {
 ### Helper Function
 
 A helper function is declared as `function {name}() {}`.
-Helper functions contain expressions and statements that can compute values.
-Helper functions that cannot be executed directly. Helper functions must be called by other functions.
+Helper functions contain expressions and statements that can compute values,
+however helper functions cannot produce `records`.
```

Comment:
Can they take records? If they don't, we might say 'cannot consume or produce'. If they do, the text is good as is.